src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { TODO } from '../../types/todo';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+import { USER } from '../../types/User';
+
+type Props = {
+  todo: TODO;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={usersFromServer.find(u => todo.userId === u.id)} />
```

Comment:
The `find` method may return `undefined` if no user with `todo.userId` is found. Ensure that the `UserInfo` component can handle an `undefined` user or provide a fallback mechanism.